java/backup-restore/pom.xml

```diff
@@ -0,0 +1,153 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.instaclustr.cassandra-operator</groupId>
+        <artifactId>operator-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>backup-restore</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <azure-storage.version>8.3.0</azure-storage.version>
+        <google-cloud-storage.version>1.84.0</google-cloud-storage.version>
+        <aws-java-sdk.version>1.11.599</aws-java-sdk.version>
+        <jakarta-el.version>3.0.2</jakarta-el.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-bom</artifactId>
+                <version>${aws-java-sdk.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.instaclustr.cassandra-operator</groupId>
+            <artifactId>operations</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>com.instaclustr.cassandra-operator</groupId>
+            <artifactId>jmx</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.inject.extensions</groupId>
+            <artifactId>guice-throwingproviders</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.inject.extensions</groupId>
+            <artifactId>guice-assistedinject</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.datastax.cassandra</groupId>
+            <artifactId>cassandra-driver-core</artifactId>
```

Comment:
can we make sure we are explicitly versioning our dependencies, otherwise the build may randomly break when the latest version is upgraded

Reply:
yes, it is versioned, that dep (with version) is just in `dependencyManagement` in parent so there is not a need to have it with versions here

Reply:
Awesome!!!